website/docs/cloud-docs/integrations/kubernetes/setup.mdx

```diff
@@ -0,0 +1,64 @@
+---
+page_title: Terraform Cloud Operator for Kubernetes setup instructions
+description: >-
+  Install and configure the Terraform Cloud Operator for Kubernetes.
+---
+
+# Terraform Cloud Operator for Kubernetes Setup Instructions
+
+Using the Terraform Cloud Operator for Kubernetes' CustomResourceDefinition (CRD), users can dynamically create Terraform Cloud workspaces using a Terraform configuration from a git repository or from the [Terraform Registry](https://registry.terraform.io/), populate variables, and perform Terraform runs to provision infrastructure. You can install the operator with the official HashiCorp Helm chart.
+
+## Prerequisites
+
+All Terraform Cloud users can use the Terraform Cloud Operator for Kubernetes. Some features of Terraform Cloud that are [limited to certain tiers](/terraform/cloud-docs/overview) aren't available to the Terraform Cloud Operator for Kubernetes unless you've purchased the corresponding tier.
+
+## Terraform Cloud Operator for Kubernetes
```

Comment:
I think we can remove this header, it doesn't add anything, and then we can boost all the below steps to h2s so they show up in the right navigation TOC
```suggestion
```